src/System/Wlog/Launcher.hs

```diff
@@ -82,7 +81,7 @@ setupLogging mTimeFunction LoggerConfig{..} = do
     liftIO $ setPrefix _lcLogsDirectory
     processLoggers mempty _lcTree
   where
-    handlerPrefix = _lcLogsDirectory ?: "."
+    handlerPrefix = fromMaybe "." _lcLogsDirectory
```

Comment:
If you're interested there's pending issue about `?:` operator in `universum`: https://github.com/serokell/universum/issues/15

You can comment on it or share some other feedback :)